### PR TITLE
Patch services.yml after flashing the image to work around install deadlock

### PR DESCRIPTION
**Pull request by alexAubin**
## Problem

It was reported on the mailing-list that there's currently an issue with the install. The install get stuck during the postinstall where it attempts to start the firewall.

## Explanation

The lock system of YunoHost was recently changed and we had a few complicated issue with that. Basically systemctl makes things not trivial to know which process launched the start/stop/restart/... (the PPID is 1), which complicates the lock handling.

Anyway, with respect to the firewall everything should be fine as long as a new line in `/etc/yunohost/services.yml`, `need_lock:true` is present for the yunohost-firewall service.

But, since the current Internet Cube images are not up to date, yunohost packages get upgraded during the install process. Naively, this shouldn't bring any trouble... but it turns out that the `/etc/yunohost/services.yml` file is not properly upgraded in that case (it's a weird situation for the package, because it's an upgrade in between the pre-installation and post-installation, and not handled correctly).

So basically `services.yml` is not updated properly, therefore the lock is not given/taken to/by the `systemctl yunohost-firewall start`, and the Cube gets stuck

## Solution

There are several possible solution for this, such as upgrading the Cube images, or fixing the upgrade process in YunoHost. But the process is a bit complicated / heavy and takes time.

Instead, a simple (dirty?) hack is to add the proper line in `services.yml` after flashing the image, similarly to the way the hypercube is added.

Hence the proposed patch :wink:

### CODE REVIEW

**Comment by sbadia** [comment]
hi @alexAubin thanks for this patch !
It's possible to rebase your pr ?

Thanks in advance

**Comment by alexAubin** [comment]
Yes, just did the rebase and fixed the small conflict :wink: 

Though maybe it should be discussed with @keomabrun or other people (?) to know what's the ETA of for new images, which should be a better fix for this that this ~dirty short-term hack, I don't know :thinking:  ...

**Comment by sbadia** [comment]
Indeed ;)

**Comment by sbadia** [comment]
Updated on repo.labriqueinter.net !
